src/cecs277TermProject/Room.java

```diff
@@ -1,20 +1,24 @@
 package cecs277TermProject;
 
-public abstract class Room {
+public class Room {
     protected int roomNumber;
     protected RoomType type;
-    protected double baseRate;
```

Comment:
What happen to `QueenRoom`, `KingRoom`, etc. if you remove this field? I thought it would give you error.